riff-raff/app/conf/context.scala

```diff
@@ -219,6 +219,26 @@ class Config(configuration: TypesafeConfig, startTime: DateTime) extends Logging
     }
   }
 
+  object management {
+    object aws {
+      // TODO read config from more generic location
+      lazy val regionName: String = getStringOpt("scheduledDeployment.aws.region").getOrElse("eu-west-1")
+
+      // For compatibility reasons this need to use the old AWS SDK
+      lazy val snsClient: AmazonSNSAsync = AmazonSNSAsyncClientBuilderV1.standard()
```

Comment:
I'm assuming it would be a fair amount of refactoring to use v2 here instead?

Reply:
Good question. I just copy pasted this from [above](https://github.com/guardian/riff-raff/blob/13fdb406e8ea032f0a3843682abc7be4e84e5aa8/riff-raff/app/conf/context.scala#L95-L104). It looks like Anghammarad wants a V1 client, so yes, sadly a fair amount of refactoring would be needed 😞 .